homeassistant/components/poolsense/sensor.py

```diff
@@ -0,0 +1,112 @@
+"""Sensor platform for the PoolSense sensor."""
+import logging
+
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+from . import get_coordinator
+from .const import ATTRIBUTION
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS = {
+    "Chlorine": {"unit": "mV", "icon": "mdi:pool", "name": "Chlorine"},
+    "pH": {"unit": "", "icon": "mdi:pool", "name": "pH"},
+    "Battery": {"unit": "%", "icon": "mdi:battery", "name": "Battery"},
```

Comment:
battery should use device classes rather than setting icon.

water temp could be temperature class.

last seen could be a timestamp as well.

https://developers.home-assistant.io/docs/entity_sensor/#available-device-classes

Reply:
Did not know about this. Thank you.